assimilation_code/modules/assimilation/assim_tools_mod.f90

```diff
@@ -988,9 +965,8 @@ subroutine obs_increment(ens_in, ens_size, obs, obs_var, obs_kind, obs_inc, &
 bounded_below = .false.;      lower_bound = 0.0_r8
 bounded_above = .false.;      upper_bound = 0.0_r8
```

Comment:
```
bounded_below = .false.;      lower_bound = 0.0_r8
bounded_above = .false.;      upper_bound = 0.0_r8
```

The bounds are intent(out) from `obs_inc_info` so misleading (and not necessary) to set them here.